app/assets/javascripts/remote_consoles/fullscreen_btn.js

```diff
@@ -0,0 +1,29 @@
+//= require jquery
```

Comment:
You don't need the jquery here, we're including it explicitly later. Also rename this file to something more "generic", like `remote_console.js` and move it one folder up.